indico/modules/events/papers/client/js/components/Paper.jsx

```diff
@@ -59,3 +105,90 @@ Paper.propTypes = {
   eventId: PropTypes.number.isRequired,
   contributionId: PropTypes.number.isRequired,
 };
+
+function PaperSteps({
+  isInFinalState,
+  isRejected,
+  hasEditable,
+  // eslint-disable-next-line no-shadow
+  editableURL,
+  fileTypes,
+  uploadableFiles,
+  loading,
+}) {
+  const {event, contribution} = useSelector(getPaperDetails);
+
+  if (loading) {
+    return (
+      <Placeholder fluid style={{margin: '2em 0'}}>
+        <Placeholder.Header image>
+          <Placeholder.Line />
+          <Placeholder.Line />
+        </Placeholder.Header>
+      </Placeholder>
+    );
+  }
+
+  return (
+    <Step.Group fluid>
+      <Step active={!isInFinalState} completed={isInFinalState}>
+        <Icon name="copy outline" />
+        <Step.Content>
+          <Step.Title>
+            <Translate>Peer Reviewing</Translate>
+          </Step.Title>
+          <Step.Description style={{marginBottom: 0}}>
+            {isInFinalState ? (
+              <Translate>The paper was reviewed</Translate>
+            ) : (
+              <Translate>Your paper is under review</Translate>
+            )}
+          </Step.Description>
+        </Step.Content>
+      </Step>
+      <Step active={isInFinalState} completed={hasEditable} disabled={isRejected}>
+        <Icon name="pencil alternate" />
+        <Step.Content>
+          <Step.Title>
+            {hasEditable || !isInFinalState || isRejected ? (
+              <Translate>Submit for Editing</Translate>
+            ) : (
+              <EditableSubmissionButton
```

Comment:
"Editing" should be mentioned here somewhere because right now we don't once the button is displayed:

![image](https://user-images.githubusercontent.com/179599/85014948-728fdc80-b167-11ea-9c9b-1a59605a925d.png)

Reply:
I tried to, the problem is, adding a title near the button completely breaks the positioning. Even if I try to make the button smaller, either the whole container will expand or they will seem out of margin. Perhaps changing the text in button?

Reply:
That's what I had in mind ;)